From ae97e41eb89ddb3b4021a93c8c6706d18cc5fc70 Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Wed, 27 Mar 2013 10:25:52 -0700
Subject: [PATCH 25/96] HBASE-7277 Thrift default JMX port should be 10103 instead of 8093

Reason: Bug
Author: Jimmy Xiang
Ref: CDH-9320
(cherry picked from commit ebbd679b964900bf4be4f5669ce898eef16c25d2)
---
 bin/hbase-config.sh |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/bin/hbase-config.sh b/bin/hbase-config.sh
index c4a95ca..c5303e8 100644
--- a/bin/hbase-config.sh
+++ b/bin/hbase-config.sh
@@ -75,9 +75,9 @@ HBASE_CONF_DIR="${HBASE_CONF_DIR:-$HBASE_HOME/conf}"
 HBASE_REGIONSERVERS="${HBASE_REGIONSERVERS:-$HBASE_CONF_DIR/regionservers}"
 # List of hbase secondary masters.
 HBASE_BACKUP_MASTERS="${HBASE_BACKUP_MASTERS:-$HBASE_CONF_DIR/backup-masters}"
-# Thrift jmx opts
+# Thrift JMX opts
 if [ -z "$HBASE_THRIFT_JMX_OPTS" ]; then
-  HBASE_THRIFT_JMX_OPTS="$HBASE_JMX_OPTS -Dcom.sun.management.jmxremote.port=8093"
+  HBASE_THRIFT_JMX_OPTS="$HBASE_JMX_OPTS -Dcom.sun.management.jmxremote.port=10103"
 fi
 # Thrift opts
 if [ -z "$HBASE_THRIFT_OPTS" ]; then
-- 
1.7.0.4

